database/migrations/000106_ruletype_evaluation_failure_message.down.sql

```diff
@@ -0,0 +1,15 @@
+-- Copyright 2024 Stacklok, Inc
```

Comment:
I just noticed the title file should be `000105_ruletype_evaluation_failure_message.down.sql` (105 not 106)

Reply:
Ah, good catch